invokeai/app/invocations/flux_denoise.py

```diff
@@ -87,6 +92,13 @@ class FluxDenoiseInvocation(BaseInvocation, WithMetadata, WithBoard):
         description="The guidance strength. Higher values adhere more strictly to the prompt, and will produce less diverse images. FLUX dev only, ignored for schnell.",
     )
     seed: int = InputField(default=0, description="Randomness seed for reproducibility.")
+    controlnet: FluxControlNetField | list[FluxControlNetField] | None = InputField(
```

Comment:
Slightly annoying to not call this `control` since that is what the SD models use. Just adds a conditional to the graph building, but may be nice to keep it consistent.

Reply:
Tidied this in [b1567fe](https://github.com/invoke-ai/InvokeAI/pull/7070/commits/b1567fe0e4d1ad462758d44802d082f9c0c6e7dc)